README.md

```diff
@@ -67,3 +67,20 @@ See the [Porting Guide](PORTING.md).
 
 ## Building
 See the [Instructions](BUILDING.md).
+## Legal
+Quantii is licensed under the [GNU Lesser General Public License v3.0](LICENSE).
+If the LICENSE file is missing, it can be found [here](https://www.gnu.org/licenses/lgpl-3.0.en.html).
+
+Until further notice, the Quantii project may not be used by any person or entity
+with the intent of commercial or monetary gain without the explicit written consent from any of the following people:
```

Comment:
I'm doubtful on how legally binding this is.  I think for a better chance (without a lawyers advice) this could be added to a TERMS.md file as well as add to the above license text to say "subject to the conditions specified in TERMS.md".